pkg/virtctl/usbredir/usbredir_default.go

```diff
@@ -31,18 +31,29 @@ import (
 )
 
 func Run(cmd *cobra.Command, args []string) error {
-	if _, err := exec.LookPath(usbredirClient); err != nil {
+	disableLaunch := cmd.Flags().Changed(optDisableClientLaunch)
+	if !disableLaunch && len(args) != 2 {
+		cmd.Usage()
```

Comment:
nit: Might be okay to not call `cmd.Usage()`.

Reply:
I meant to only return `Missing argument`. IIRC in most other scenarios it does not print the usage string again, e.g. when cobra.RangeArgs(1, 2) fails.